src/services/pull/useSingularImpactedFileComparison.tsx

```
@@ -166,11 +167,8 @@ export function useSingularImpactedFileComparison({
           )
         }
 
-        return Promise.reject({
-          status: 404,
-          data: {},
-          dev: 'useSingularImpactedFileComparison - 404 missing data',
-        } satisfies NetworkErrorObject)
+        // we can set to null, and use the error display message we currently have, rather than throwing
```

not sure I understand what's going on here. Can you provide some more context?

---

so before we were throwing an error here rather than returning anything. However, in all locations where this is being used, we're checking to ensure data exists and is not a nullish value, and if it is, we display an error message.